packages/build-info/src/detect-build-system.ts

```diff
@@ -0,0 +1,154 @@
+import { readFileSync } from 'fs'
+import path from 'path'
+
+import { findUpSync } from 'find-up'
+import { PackageJson } from 'read-pkg'
+
+export type BuildSystem = {
+  name: string
```

Comment:
should we already extend this with a optional icon? like framework info is doing?